UnityXrefMaps/XrefMapReference.cs

```diff
@@ -40,7 +41,7 @@ public sealed class XrefMapReference
         /// <summary>
         /// Gets if this <see cref="XrefMapReference"/> is valid or not.
         /// </summary>
-        public bool IsValid => !commentId.Contains("Overload:");
+        public bool IsValid => true;
```

Comment:
The idea here was to not generate reference to pointers. Do you think they should be generated?